firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -76,7 +76,8 @@ void alignAxis(uint8_t motor[], uint8_t alignmentCode)
   int eighthStepsPerPulse;
   int i;
   uint8_t endstopPin;
-
+  uint16_t tempAlignWholeSteps, j;
```

Comment:
You can reuse `i` instead of making a new var, just make sure to reset it to 0 (which you are already doing in the for loop initialization).